Apps/W1/Shopify/app/src/Payments/Codeunits/ShpfyPayments.Codeunit.al

```diff
@@ -203,4 +203,48 @@ codeunit 30169 "Shpfy Payments"
             end;
         end;
     end;
+
```

Comment:
This request is missing pagination (https://shopify.dev/docs/api/usage/pagination-rest).
Check out the other requests of the Shopify App